app/controllers/application_controller.rb

```diff
@@ -62,4 +62,8 @@ def render_unauthorized
     headers['WWW-Authenticate'] = 'Token realm="Application"'
     render json: 'Not Authorized', status: 401
   end
+
+  rescue_from ActionController::ParameterMissing do
+    render json: { error: 'parameter missing' }, status: :bad_request
+  end
```

Comment:
In terms of location, this needs to be after the other `rescue_from` in order to be handled properly. I went ahead and put it underneath all of the other supporting code for handling the first `rescue_from`, but I can see the case for having it next to the first one too.